parser.go

```diff
@@ -730,7 +757,11 @@ func (p *parser) handleExplanation(t *token) (string, error) {
 		return "", SyntaxError{t, newInvalidDomainError(domain)}
 	}
 
-	txts, _, err := p.resolver.LookupTXT(NormalizeFQDN(domain))
+	txts, extras, err := p.resolver.LookupTXT(NormalizeFQDN(domain))
+	if extras.Void() {
+		p.fireVoidLookup()
+	}
+
```

Comment:
We did not have previously fired the Explanation event to the listener. Fire void lookup without it will mess up the walker state

Reply:
updated